.env.example

```diff
@@ -22,6 +22,8 @@ CALCOM_LICENSE_KEY=
 
 # - DATABASE ************************************************************************************************
 DATABASE_URL="postgresql://postgres:@localhost:5450/calendso"
+# So we can use Prisma Data Proxy (https://www.prisma.io/docs/data-platform/data-proxy/use-data-proxy)
```

Comment:
Perhaps a better description (what can a self hoster use this for if they want to?, can they, should they?)

Reply:
The link should be enough? Or do you mean specific benefits like cold-boots?

Reply:
Added a better explainer